nikola/post.py

```diff
@@ -745,11 +745,16 @@ def section_color(self, lang=None):
 
     def section_link(self, lang=None):
```

Comment:
Maybe we should deprecate this function and encourate people to use `site.link()` instead. A `Post` object should know as little as possible about what other plugins are producing (well, except the `pages` and the `posts` plugins).